python/ray/tests/test_basic.py

```diff
@@ -1111,7 +1113,11 @@ def test_import_ray_does_not_import_grpc():
     assert "grpc" not in sys.modules
 
     # Load grpc back so other tests will not be affected
-    import grpc  # noqa: F401
+    try:
+        import grpc  # noqa: F401
+    except ImportError:
+        # It's ok if we don't have grpc installed.
```

Comment:
can you add `assert os.environ.get("RAY_MINIMAL") == 1` here?

Reply:
I don't see the point. It's possible someone has grpc installed on their own, and installed minimal ray, and run this test.